tests/tests_geomstats/test_discrete_curves.py

```diff
@@ -79,6 +77,7 @@ def test_belongs(self):
         result = self.space_curves_in_sphere_2d.belongs(curve_ab)
         self.assertTrue(gs.all(result))
 
+    @geomstats.tests.np_autograd_and_torch_only
```

Comment:
What changed for tensorflow? (it seems that it was working before?)

Reply:
The L2 metric from Landmarks has been replaced by L2CurvesMetric, and with tensorflow, there is an error occurring in `gs.vectorize` used in the method `pointwise_inner_products`. I'm not sure why.

Reply:
Ok!